src/audio/CMakeLists.txt

```diff
@@ -49,6 +49,7 @@ if(NOT BUILD_HOST)
 	if(CONFIG_COMP_MUX)
 		add_local_sources(sof
 			mux.c
+			mux_generic.c
```

Comment:
I would say we call this muxdemux.c since it's more powerful and complex than a simple stream 1:1 mux where one source is routed to one sink.